components/footer.tsx

```diff
@@ -1,23 +1,37 @@
 /* eslint-disable @next/next/no-img-element */
 import React, { FunctionComponent } from "react";
 import styles from "../styles/footer.module.css";
+import ScrollUp from "./scrollUp";
+import { Parallax } from "react-scroll-parallax";
 
 const Footer: FunctionComponent = () => {
   return (
-    <footer className={styles.footer}>
-      <div className="mb-4">
-        Powered by&nbsp;<strong>Starknet</strong>
-      </div>
-      <div className="flex text-s">
-        <a className="underline" href="/pdfs/PrivacyPolicy.pdf">
-          Privacy policy
-        </a>
-        &nbsp;-&nbsp;
-        <a className="underline" href="/pdfs/Terms.pdf">
-          Terms of use
-        </a>
-      </div>
-    </footer>
+    <>
+      <Parallax speed={-20}>
+        <img
+          src="/leaves/lilLeaf02.svg"
+          alt="leaf"
+          className="absolute right-10 mr-16"
+          width={20}
+          height={20}
+        />
+      </Parallax>
+      <footer className={styles.footer}>
+        <ScrollUp />
+        <div className="mb-4">
+          Powered by&nbsp;<strong>Starknet</strong>
+        </div>
+        <div className="flex text-s">
+          <a className="underline" href="/pdfs/PrivacyPolicy.pdf">
```

Comment:
Can you make this with `window.open()`, it will make users don't move from the website

Reply:
Why not using target="_blank" ? The result would be the same but I think this is better for accessibility and it is pure html